src/components/Create/components/RewardsList/AddEditRewardModal.tsx

```diff
@@ -236,19 +282,108 @@ export const AddEditRewardModal = ({
+                value={isReservingNfts}
+                onChange={setIsReservingNfts}
+                label={
+                  <span className="flex items-center gap-2">
+                    <Trans>Reserve NFTs</Trans>
+                    <Tooltip
+                      className="cursor-help text-grey-500 dark:text-grey-300"
+                      title={t`If Reserved NFTs are set, the first reserved NFT from the tier will be distributable once the tier receives its first mint.`}
+                    >
+                      <QuestionCircleOutlined />
+                    </Tooltip>
+                  </span>
+                }
+              />
+            </Form.Item>
+            {isReservingNfts && (
+              <>
+                <span className="mb-4 flex flex-col gap-2 whitespace-nowrap md:flex-row md:items-center">
+                  <Trans>Reserve 1 NFT of every</Trans>
+                  <div className="max-w-[78px]">
+                    <Form.Item
+                      className="mb-0"
+                      name="nftReservedRate"
+                      rules={[
+                        inputMustExistRule({ label: t`Reserved rate` }),
+                        inputIsIntegerRule({
+                          stringOkay: true,
+                        }),
+                      ]}
+                    >
+                      <FormattedNumberInput min={2} placeholder="0" />
+                    </Form.Item>
+                  </div>{' '}
+                  <Trans>NFTs minted for:</Trans>
+                </span>
+                <Form.Item
+                  className="mb-8"
+                  name="beneficiary"
+                  label={t`Ethereum wallet address`}
+                  extra={t`Reserved NFTs will be sent to this address once minted.`}
+                  rules={[
+                    inputMustExistRule({ label: t`Wallet address` }),
+                    inputMustBeEthAddressRule({ label: t`Wallet address` }),
+                  ]}
+                >
+                  <EthAddressInput />
+                </Form.Item>
+              </>
+            )}
+            <Form.Item
+              name="votingWeight"
+              label={t`Voting weight`}
+              tooltip={t`Give this NFT a voting weight to be used for on-chain governance. The number you set is only used in relation to other NFTs in this collection.`}
+              extra={
+                <Trans>
+                  If you use the default governance option (no governance), the
+                  voting weight will still be accessible on the blockchain for
+                  use in snapshot strategies or any other desired purpose.
+                </Trans>
+              }
+              rules={[
+                inputIsIntegerRule({
+                  label: t`Voting weight`,
+                  stringOkay: true,
+                }),
+              ]}
+            >
+              <FormattedNumberInput placeholder="300" />
```

Comment:
I think we agreed to scrap placeholders totally a while ago, especially in cases of number values like this. 

However, I do think we need more of an explanation as to what value belongs here. Weight of what? 

I think after we add more info about this, there'll be enough text to warrant outsourcing most of this field's text to a docs page (not sure if one already exists). I think in the end just an extra with...
 - `Give this NFT a voting weight to be used for on-chain governance. <b>Learn more</b>.`
 
... will be all that is necessary. And then move everything else into the 'Learn more' docs page. Wdyt?

Reply:
Yep, strong agree with this one

Reply:
I think we should have a greater discussion about this too. Unsure if it is blocking of this PR as we can make the change once we port over to a docs link too

Reply:
Created issue for the docs page: https://github.com/jbx-protocol/juice-interface/issues/2900

But I think really think we should scrap the placeholder, it doesn't really make sense to have here regardless.

Reply:
Ah yep - willr emove placeholder